### DISCUSSION TITLE
LoadResPicture improvements

### DISCUSSION THREAD

**Discussion by WaynePhillipsEA** [twinbasic/twinbasic#369] [General]
As per the changelog, v0.10.4628 adds a new enum value for LoadResPicture:  vbResBitmapFromIcon.  As the name suggests, this allows ICO resources to be loaded as bitmap-based IPictureDisp objects rather than icon-based IPictureDisp objects.

This is useful when dealing with the Office Ribbon, as it doesn't support icon-based IPictureDisp objects.

LoadResPicture also now offers optional Width and Height parameters, which the system will use for resizing and/or more importantly for determining which icon to load when an ICO file contains multiple images.

https://marketplace.visualstudio.com/items/twinbasic.twinbasic/changelog

**Comment by Kr00l** [comment]
Cool, but I don't get this to work. Is there a sample project available ?

Also which background color will be applied for that ICO which is then bitmap-based ?

**Comment by Kr00l** [comment]
Addition: the best would be to create a 32 bit alpha bitmap, thus having the same transparency as the icon.

**Comment by Kr00l** [comment]
I still don't get this vbResBitmapFromIcon to work. :)

**Reply by Kr00l** [reply]
> I still don't get this vbResBitmapFromIcon to work. :)

Reminder. ;-)

**Reply by WaynePhillipsEA** [reply]
It's (now) on my list for this coming week :)
